openssl/aes_test.go

```diff
@@ -35,7 +35,6 @@ func TestNewGCMNonce(t *testing.T) {
 	if err != nil {
 		t.Errorf("expected no error for standard tag / nonce size, got: %#v", err)
 	}
-	c.finalize()
```

Comment:
Should we add a test to explicitly test the finalizer? (`runtime.SetFinalizer(c, nil)` and run it manually in the test body?)

Reply:
Done!